tests/cupy_tests/core_tests/test_ndarray_adv_indexing.py

```diff
@@ -146,35 +150,33 @@ def test_adv_getitem(self, xp, dtype):
     {'shape': (2, 3, 4), 'indexes': [[1]]},
     {'shape': (2, 3, 4), 'indexes': [[1, 1]]},
     {'shape': (2, 3, 4), 'indexes': [[1], [1]]},
-    {'shape': (2, 3, 4), 'indexes': [[1, 1], 1]},
-    {'shape': (2, 3, 4), 'indexes': [[1], slice(1, 2)]},
-    {'shape': (2, 3, 4), 'indexes': [[[1]], slice(1, 2)]},
 )
-@testing.gpu
-class TestArrayAdvancedIndexingGetitemDeprecated:
+@testing.with_requires('numpy>=1.23')
+class TestArrayAdvancedIndexingGetitemParametrized2:
 
     @testing.for_all_dtypes()
     @testing.numpy_cupy_array_equal()
     def test_adv_getitem(self, xp, dtype):
-        with testing.assert_warns(FutureWarning):
-            a = testing.shaped_arange(self.shape, xp, dtype)
-            return a[self.indexes]
+        a = testing.shaped_arange(self.shape, xp, dtype)
+        return a[self.indexes]
 
 
 @testing.parameterize(
-    {'shape': (0,), 'indexes': True},
-    {'shape': (0,), 'indexes': (True,)},
-    {'shape': (0,), 'indexes': numpy.ones((), dtype=numpy.bool_)},
-    {'shape': (0,), 'indexes': numpy.zeros((), dtype=numpy.bool_)},
```

Comment:
These tests were split from `TestArrayAdvancedIndexingGetitemParametrized` in #1503 for NumPy 1.13 support, so I've made them back there in this PR.